tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/AsynchronousTest.java

```diff
@@ -101,33 +91,31 @@ public void testAsyncIsFinished() {
      */
     @Test
     public void testClassLevelAsyncIsNotFinished() {
-        Future<Connection> future = null;
-        try {
-            future = clientClass.service();
-        }
-        catch (InterruptedException e) {
-            throw new AssertionError("testAsync: unexpected InterruptedException calling service");
-        }
-
-        Assert.assertFalse(future.isDone());
-
+        CompletableFuture<Void> waitingFuture = newWaitingFuture();
+        CompletionStage<Connection> resultFuture = clientClass.serviceCS(waitingFuture);
+        Future<Connection> future = resultFuture.toCompletableFuture();
+        await().atMost(400, TimeUnit.MILLISECONDS).untilAsserted(()-> Assert.assertFalse(future.isDone()));
     }
 
     /**
      * Test that the future returned by calling a method in an asynchronous class is done if called after waiting enough time to end the operation
      */
     @Test
     public void testClassLevelAsyncIsFinished() {
-        Future<Connection> future = null;
-        try {
-            future = clientClass.service();
-            Thread.sleep(1500);
-        }
-        catch (InterruptedException e) {
-            throw new AssertionError("testAsync: unexpected InterruptedException calling service");
-        }
-
-        Assert.assertTrue(future.isDone());
+        CompletableFuture<Void> waitingFuture = newWaitingFuture();
+        CompletionStage<Connection> resultFuture = clientClass.serviceCS(waitingFuture);
```

Comment:
Same here, should call `service()`, not `serviceCS()`.

Reply:
done